### ISSUE TITLE
statiCards shown incomplete when app is started

### ISSUE THREAD

**Issue by BrianCollinss** [dreamRs/shinyWidgets#406]
You can see the issue here: http://incentiv8.opticane.net/

When the website is open and app is run for the first time (without changing any of the options), statiCards are not shown properly:

![image](https://user-images.githubusercontent.com/39573622/130723575-b785585f-8906-4968-b90e-7b57b528bc5b.png)

But when I change one of the options and statiCards are updated, they appear fineL

![image](https://user-images.githubusercontent.com/39573622/130723632-39b1f732-93f1-4757-9862-1e472299a7e3.png)

**Comment by pvictor** [comment]
Hello,

Do you use `uiOutput` / `renderUI` to display cards ? It seems that it can cause this issue.
Have you tried to declare the card in your UI only once with : 

```r
statiCard(
  93, "Animated card", icon("users"),
  background = "deepskyblue",
  color = "white",
  animate = TRUE,
  id = "card1" # <----- set an ID
)
```

And then only updating the value in the server using card's ID with: 

```r
updateStatiCard(
  id = "card1", # <----- use an ID
  value = 58 # new value
)
```

Victor

**Comment by BrianCollinss** [comment]
Yes, I use `uiOutput`. Is there going to be any update to fix the issue? As I have many StatiCards and I also need to determine their colors based on the value, it is much easier not to use `updateStatiCard`.
